.github/workflows/build.yml

```diff
@@ -0,0 +1,54 @@
+name: Build
+
+on: push
+
+jobs:
+  windows_build:
+    name: Windows Build
+    runs-on: windows-latest
+    env:
+      BUILD_TYPE: Release
```

Comment:
This environment variable seems to be unused, maybe consider adding `-DCMAKE_BUILD_TYPE=${{ env.BUILD_TYPE }}`` or just directly `-DCMAKE_BUILD_TYPE=Release` to the cmake init command